entity-framework/docfx.json

```diff
@@ -67,14 +67,11 @@
       "searchScope": [
         "Entity Framework"
       ],
-      "manager": "divega",
       "ms.topic": "conceptual",
       "audience": "developer",
-      "ms.reviewer": "divega",
       "ms.prod": "entity-framework",
       "ms.tgt_pltfrm": "dotnet",
       "ms.devlang": "dotnet",
-      "ms.author": "divega",
```

Comment:
The other two are fine to remove, but for this one you need to someone. Pick someone and remember to update when you have a new PM?

Reply:
https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/docfx.json#L49-L61

I saw it missing in aspnetcore docs so thought it is not needed. Is it GitHub handle or Microsoft alias?

Reply:
They're probably being set on an individual-topic basis: https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/data/entity-framework-6.md

Microsoft alias

Reply:
But I'll go check our metadata report to see if anything is showing up there as missing

Reply:
@mairaw - Necessary or not? I will put it back if required.

Reply:
Putting it back.

Reply:
Yes, it's required.

Reply:
Also, it's always good to have a default and then you can override in individual articles.

Reply:
You'll probably want to remove them from individual articles as well: https://github.com/aspnet/EntityFramework.Docs/search?q=divega&unscoped_q=divega

Reply:
We also contains names of previous author in a lot of EF6 article too including Rowan, let me discuss with @ajcvickers on how we plan to deal with such references. So far we have been updating the author name whenever we update any article.